glusterd2/commands/global/getglobaloptions.go

```diff
@@ -0,0 +1,45 @@
+package globalcommands
+
+import (
+	"net/http"
+
+	"github.com/gluster/glusterd2/glusterd2/cluster"
+	restutils "github.com/gluster/glusterd2/glusterd2/servers/rest/utils"
+	"github.com/gluster/glusterd2/pkg/api"
+)
+
+func getGlobalOptionsHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	c, _ := cluster.GetCluster()
```

Comment:
Don't ignore error. 

Ignore only specific error if required. Error may be due to etcd connectivity.

Reply:
Yes. Missed the error handling here.